content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md

```diff
@@ -0,0 +1,65 @@
+#
+# RapidPro CHT APK with FCM configuration
+
+RapidPro - CHT uses your Android phone to send and receive messages on your behalf.
+
+Due to restrictions Android has on applications that send SMS messages, we cannot distribute the RapidPro - CHT application through the Play Store. You will need to download the application manually and install it using the steps below.
+
+
+
+1. Download App
+
+    On your Android device, open your browser and download the app by entering the URL: <code>[https://rapidpro.app.medicmobile.org/android/](https://rapidpro.app.medicmobile.org/android/)</code>
+
+
+
+2. Launch the RapidPro - CHT installer
+
+    Open the notifications shade by dragging from the top and select the downloaded file.
+
+
+![download](rapidpro/apk_download.png "image_tooltip")
```

Comment:
The images are not getting rendered. Have a look at the [documentation on image processing](https://gohugo.io/content-management/image-processing/) to guide you on how to add images.